docs/README.md

```diff
@@ -0,0 +1,325 @@
+
+EVE is *not* a traditional operating system in that it doesn't support applications
+represented as executable files and [POSIX processes](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_295)
+and expects all its application to be represented as Edge Containers.
+
+Think of it this way: Android has [APKs](https://en.wikipedia.org/wiki/Android_application_package),
+Docker has [OCI Containers](https://www.opencontainers.org) - EVE has Edge Containers.
+
+You can read a detailed specification of [Edge Containers](ECOS.md) and you should
+expect them to be available for broad Linux Foundation adoption once we are ready
+for formal submission of a self-contained specification.
+
+## Security
+
+What makes EVE a secure-by-design system is that it has been developed from the
+ground up with security in mind. EVE doesn't rely on 3rd party or add-on components
+to provide trust in the system, but rather offers a set of well-defined principles
+on which a trust model can be built.
+
+We have made an effort to provide users of EVE with a system that is both
+practically secure and can be deployed in a zero-touch fashion. In order to achieve
+our goals we put forward a [few guiding principles](SECURITY.md).
+
+## EVE Controller
+
+Each Edge Node with EVE running on it can participate in sophisticated
+[Edge Computing scenarious](https://www.zdnet.com/article/10-scenarios-where-edge-computing-can-bring-new-value/)
+ranging from simple [data collection](https://blog.equinix.com/blog/2017/11/29/how-iot-data-collection-and-aggregation-with-local-event-processing-work/)
+all the way to [Edge Analytics and AI](https://www.datanami.com/2019/02/04/exploring-artificial-intelligence-at-the-edge/).
+While EVE provides an incredible amount of functionality to support these use
+cases, it needs to be told exactly how to orchestrate these functional building blocks
+by a *Controller*. In other words, without Controller telling a fleet of Edge
+Nodes what to do, EVE by itself may be capable, but it practically useless.
+
+This is a fundamental point of EVE's design: an end user (lets call her Diane
+The DevOps) trying to implement an Edge Computing scenario never communicates
+directly with a running EVE instance. There are no user names, passwords or a
+concept of "logging in" when it comes to EVE. All interactions that Diane will
+have are going to be through the EVE Controller. Diane expresses her overall
+intent to a controller using a Controller API and it is then the job of a
+controller to build an execution plan for a fleet of Edge Nodes running EVE.
+When each instance of EVE contacts the controller using EVE API it will receive
+its own portion of the overall execution plan (note that EVE API communications
+are  always initiated by EVE, never by the Controller):
+
+![EVE Controller](pics/EVEController.png)
+
+A good way to think about this is that Controller is to EVE what Kubernetes is
+to Docker Engine.
```

Comment:
This is a good analogy which many will grasp right away. I am a bit concerned that it makes EVE look like, "oh, someone just trying to do k8s at the edge" (which, to some degree, it is), but EVE is much more than that. I would leave it for now, but keep an eye on the impact.

Reply:
good point, but lets leave it for now as you say